ui/src/org/pentaho/di/ui/core/database/dialog/tags/ExtTextbox.java

```diff
@@ -22,10 +22,12 @@
 
 package org.pentaho.di.ui.core.database.dialog.tags;
 
+import org.drools.lang.DRLParser.type_declaration_return;
```

Comment:
Yes, not sure how Eclipse didn't flag it, as it wasn't used anywhere.  It's been removed in the latest commit.